backend/stack.yaml

```diff
@@ -2,3 +2,8 @@ resolver: lts-17.2
 
 packages:
 - .
+
+extra-deps:
+  - aeson-options-0.1.0
+
+allow-newer: true
```

Comment:
Is it needed for `aeson-options` (that we shouldn't use because it's deprecated) or something else?

Reply:
yes, I will remove if it's needed for `aeson-casing`